src/main/java/stream/flarebot/flarebot/mod/ModlogEvent.java

```diff
@@ -0,0 +1,70 @@
+package stream.flarebot.flarebot.mod;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.User;
+import org.apache.commons.lang3.text.WordUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.Color;
+
+public enum ModlogEvent {
+
+    MEMBER_JOIN("Triggers when user joins the server.", Color.GREEN),
+    MEMBER_LEAVE("Triggers when user leaves the server.", Color.RED),
+
+    MEMBER_ROLE_GIVE("Triggers when a role is added to a user.", Color.GREEN),
+    MEMBER_ROLE_REMOVE("Triggers when a role is taken away a user.", Color.RED),
+
+    MEMBER_VOICE_JOIN("Triggers when a user joins a voice channel.", Color.GREEN),
+    MEMBER_VOICE_LEAVE("Triggers when a user leaves a voice channel.", Color.RED),
+    MEMBER_VOICE_MOVE("Triggers when a user moves to a different voice channel. Either because they moved or because they where force moved.", Color.decode("#addfe6")),
+
+    MEMBER_NICK_CHANGE("Triggers when a user changes their nick.", Color.decode("#addfe6")),
+
+    ROLE_CREATE("Triggers when a role is created.", Color.GREEN),
+    ROLE_DELETE("Triggers when a role is deleted.", Color.RED),
+    ROLE_EDIT("Triggers when a role is edited.", Color.decode("#addfe6")),
+
+    CHANNEL_CREATE("Triggers when a channel is created.", Color.GREEN),
+    CHANNEL_DELETE("Triggers when a channel is deleted.", Color.RED),
+
+    COMMAND("Triggers when a user runs a **FlareBot** command.", Color.decode("#addfe6")),
```

Comment:
Change this to FLAREBOT_COMMAND and make it the bottom one since it doesn't refer to the other "normal" discord events.